controllers/imageclusterinstall_controller.go

```diff
@@ -310,6 +313,10 @@ func (r *ImageClusterInstallReconciler) validateSeedReconfigurationWithBMH(
 	ici *v1alpha1.ImageClusterInstall,
 	bmh *bmh_v1alpha1.BareMetalHost) (ctrl.Result, error) {
 
+	if bmh.Spec.ExternallyProvisioned {
```

Comment:
maybe lets add comment that there is nothing to validate in case host is ExternallyProvisioned as it will not be inspected?